chatterbot/adapters/storage/mongodb.py

```diff
@@ -173,11 +173,11 @@ def filter(self, **kwargs):
 
         return results
 
-    def update(self, statement):
+    def update(self, statement, force = False):
```

Comment:
I agree with @karanluthra, there should be no spaces when doing assignments in method parameters.